mappings/net/minecraft/client/gui/Element.mapping

```diff
@@ -1,38 +1,148 @@
+		COMMENT @see org.lwjgl.glfw.GLFW#GLFW_KEY_Q
+		COMMENT @see org.lwjgl.glfw.GLFWKeyCallbackI#invoke(long, int, int, int, int)
 		ARG 1 chr
 		ARG 2 keyCode
 	METHOD method_25401 mouseScrolled (DDD)Z
+		COMMENT Callback for when a mouse button scroll event
+		COMMENT has been captured.
+		COMMENT
+		COMMENT @param mouseX the X coordinate of the mouse
+		COMMENT @param mouseY the Y coordinate of the mouse
+		COMMENT @param amount value is {@code > 1} if scrolled down, {@code < 1} if scrolled up
+		COMMENT @return {@code true} to indicate that the event handling is successful/valid
+		COMMENT @see net.minecraft.client.Mouse#onMouseScroll(long, double, double)
 		ARG 1 mouseX
 		ARG 3 mouseY
 		ARG 5 amount
 	METHOD method_25402 mouseClicked (DDI)Z
+		COMMENT Callback for when a mouse button down event
+		COMMENT has been captured.
+		COMMENT
+		COMMENT The button number is identified by the constants in
+		COMMENT {@link org.lwjgl.glfw.GLFW GLFW} class.
+		COMMENT
+		COMMENT @param mouseX the X coordinate of the mouse
+		COMMENT @param mouseY the Y coordinate of the mouse
+		COMMENT @param button the mouse button number
+		COMMENT @return {@code true} to indicate that the event handling is successful/valid
+		COMMENT @see net.minecraft.client.Mouse#onMouseButton(long, int, int, int)
+		COMMENT @see org.lwjgl.glfw.GLFW#GLFW_MOUSE_BUTTON_1
 		ARG 1 mouseX
 		ARG 3 mouseY
 		ARG 5 button
 	METHOD method_25403 mouseDragged (DDIDD)Z
+		COMMENT Callback for when a mouse button drag event
+		COMMENT has been captured.
+		COMMENT
+		COMMENT The button number is identified by the constants in
+		COMMENT {@link org.lwjgl.glfw.GLFW GLFW} class.
+		COMMENT
+		COMMENT @param mouseX the current X coordinate of the mouse
+		COMMENT @param mouseY the current Y coordinate of the mouse
+		COMMENT @param button the mouse button number
+		COMMENT @param deltaX the difference of the current X with the previous X coordinate
+		COMMENT @param deltaY the difference of the current Y with the previous X coordinate
+		COMMENT @return {@code true} to indicate that the event handling is successful/valid
+		COMMENT @see net.minecraft.client.Mouse#onCursorPos(long, double, double)
+		COMMENT @see org.lwjgl.glfw.GLFW#GLFW_MOUSE_BUTTON_1
 		ARG 1 mouseX
 		ARG 3 mouseY
 		ARG 5 button
 		ARG 6 deltaX
 		ARG 8 deltaY
 	METHOD method_25404 keyPressed (III)Z
+		COMMENT Callback for when a key down event has been captured.
+		COMMENT
+		COMMENT The key code is identified by the constants in
+		COMMENT {@link org.lwjgl.glfw.GLFW GLFW} class.
+		COMMENT
+		COMMENT See the difference of a key code, a scan code, and a
+		COMMENT modifier in {@link org.lwjgl.glfw.GLFWKeyCallbackI#invoke(long, int, int, int, int) GLFWKeyCallbackI}
+		COMMENT
+		COMMENT @param keyCode the key code of the event
+		COMMENT @param scanCode the scan code of the event
+		COMMENT @param modifiers a bitfield/flag describing the which modifier keys are held down
```

Comment:
Could you link to what the bitset of modifiers contains and how you know what modifiers are being used?

Reply:
Provided the link to the GLFW online doc